cmd/build/v2/build.go

```diff
@@ -111,6 +113,7 @@ func NewBuilderFromScratch(analyticsTracker analyticsTrackerInterface) *OktetoBu
 		buildEnvironments: map[string]string{},
 		Config:            getConfig(registry, gitRepo),
 		analyticsTracker:  analyticsTracker,
+		repo:              gitRepo,
```

Comment:
can we include inside `Config` this new prop instead of injecting the repo?

Reply:
cebae20a007361f6ea2f43a51e8d273c7b0e78eb